applications/StructuralMechanicsApplication/python_scripts/kratos_main_structural.py

```diff
@@ -110,9 +117,8 @@
     main_model_part.CloneTimeStep(time)
 
     if (parallel_type == "OpenMP") or (mpi.rank == 0):
-        Logger.PrintInfo("")
-        Logger.PrintInfo("STEP = "+str(main_model_part.ProcessInfo[STEP]))
-        Logger.PrintInfo("TIME = "+str(time))
+        Logger.PrintInfo("STEP", str(main_model_part.ProcessInfo[STEP]))
+        Logger.PrintInfo("TIME", str(time))
```

Comment:
Just a question, Does it work without str() function?

Reply:
Should be